### DISCUSSION TITLE
Transaction Issue

### DISCUSSION THREAD

**Discussion by marco3411** [Chia-Network/chia-blockchain#17160] [Support]
I am writing to report a problem with a recent transaction related to my wallet. The details of the transaction are as follows:

Amount: 12 XCH 
Recipient Address: xch1nrdfarwst8e5wxgj0lr8ut6n7gp752hkys9agd0x3ahshg3ja03qeh8z7t
Sending Date: December 23, 2023 11:40PM Serbia

The issues I am encountering are as follows:

Although the amount of 12 XCH has been successfully deducted from my wallet, the transaction does not appear on the xchscan.com.

Instead, I notice another transaction of 10 XCH that I did not authorize. This transaction was sent to an unknown address, not the one I specified when sending the 12 XCH.
This is causing concern, and I urgently need a resolution to this problem. Please investigate this situation and provide me with information on the status of the 12 XCH transaction.

My identification details:

Wallet Address: xch188c8an4f03n7sv08vufvt5r8kwuzsnfytclsnuur8fylecsg636s00znsh
I appreciate your prompt attention to this matter and request updates on each step of the resolution process.

Thank you in advance for your assistance.

![Screenshot_3](https://github.com/Chia-Network/chia-blockchain/assets/154734217/ba262299-f1aa-4589-82bd-cb6b77cb364e)
![Screenshot_2](https://github.com/Chia-Network/chia-blockchain/assets/154734217/e5927666-d08d-4a95-b372-639ca2e57434)

**Comment by coucou2003** [comment]
Hi , there , I have the same issue i send 8 XCH but receive  only 5 after 6 days
![Capture2](https://github.com/Chia-Network/chia-blockchain/assets/155120224/52655572-ce06-4d26-9855-17d8ac4c749e)

**Comment by BrandtH22** [comment]
Hey @marco3411 , as a heads up we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia)

As for the transaction, from what I am seeing onchain the 12xch transaction never went through and the 10xch transaction was routed back to your main xch address (https://alltheblocks.net/chia/coin/0xf4a7875a793a791a341c9358a78ce9872a6cde8f14c354b18ca0eb6469151fc8). 

What I would do is resync the wallet (gui settings / advanced / resync) and upon resyncing reattempt the original 12 xch transaction with a reasonable fee.

Let us know if any questions arise :+1:

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
